content/learn/book/ecs/exclusive-world-access/_index.md

```diff
@@ -1,14 +1,138 @@
 +++
 title = "Exclusive `World` access"
-weight = 8
+weight = 7
 template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: show how to work with a raw world
+In various places, Bevy code works directly with the `World` data, gaining exclusive, blocking access to it and allowing for arbitrary mutations.
+This is very flexible, and can be essential for advanced use cases, but prevents any other systems from running simultaneously and can often be harder to reason about and maintain.
+As a result, you should use only exclusive `World` when you have to.
+You might be working with the `World` if:
 
-TODO: discuss and demonstrate custom commands
+- you're running an exclusive system to access data in unusually broad ways (such as for saving the game or handling networking)
+- you're writing a custom command, to execute logic at the end of the system
+- you're initializing a resource using the `FromWorld` trait
+- you're working with a `NonSend` resource that cannot be sent across threads
```

Comment:
Perhaps call this `NonSend` Values, to avoid confusion with regular systems that use `NonSend`Resources and are not exclusive.
```suggestion
- you're working with `NonSend` values that cannot be sent across threads
```

Reply:
They're resources in every other respect, and are stored with other resources.

That said, I like this change as there's no blockers to us implementing NonSend components too.